tests/unittests/utils/test_sam2.py

```
@@ -0,0 +1,736 @@
+        mock_predictor.device = torch.device("cpu")
+        mock_predictor.predict.return_value = (
+            np.ones((1, h, w), dtype=np.float32),
+            np.array([0.95]),
+            None
+        )
+
+        model._load_predictor = mock.MagicMock(return_value=mock_predictor)
+
+        img = torch.ones((3, h, w))
+
+        def mock_to_abs_boxes(boxes, width, height, chunk_size=None):
```

Ruff (0.14.11)

Unused function argument: `chunk_size`

(ARG001)